app/renderer/network.html

```diff
@@ -14,14 +14,16 @@
   <body>
     <div id="content">
       <div id="picture"><img src="img/zulip_network.png" /></div>
-      <div id="title">We can't connect to this organization</div>
-      <div id="subtitle">This could be because</div>
-      <ul id="description">
-        <li>You're not online or your proxy is misconfigured.</li>
-        <li>There is no Zulip organization hosted at this URL.</li>
-        <li>This Zulip organization is temporarily unavailable.</li>
-        <li>This Zulip organization has been moved or deleted.</li>
-      </ul>
+      <div id="texts">
```

Comment:
"texts" seems like a very general selector ID.

Reply:
I agree, It was a general selector id. I changed it to 'network-error-texts' with my last commit so it would be more obvious and easy to understand.